nucypher/policy/conditions/json/json.py

```diff
@@ -0,0 +1,85 @@
+import json
+from typing import Any, Optional, Tuple
+
+from marshmallow import fields, post_load, validate
+
+from nucypher.policy.conditions.base import Condition
+from nucypher.policy.conditions.context import resolve_any_context_variables
+from nucypher.policy.conditions.exceptions import InvalidCondition
+from nucypher.policy.conditions.json.base import JSONPathField
+from nucypher.policy.conditions.json.utils import (
+    process_result_for_condition_eval,
+    query_json_data,
+)
+from nucypher.policy.conditions.lingo import AnyField, ConditionType, ReturnValueTest
+
+
+def _parse_json_data(data: Any) -> Any:
+    """
+    Parse JSON data if it's a string, otherwise return as-is.
+    """
+    if not isinstance(data, str):
+        return data
+
+    try:
+        return json.loads(data)
+    except (json.JSONDecodeError, ValueError) as e:
```

Comment:
`JSONDecodeError` is already a subclass of `ValueError`

```suggestion
    except (ValueError) as e:
```